jenkins/agent-base/Dockerfile.ubi8

```diff
@@ -10,7 +10,9 @@ ENV SONAR_SCANNER_VERSION=4.7.0.2747 \
     HELM_PLUGIN_DIFF_VERSION=3.6.0 \
     HELM_PLUGIN_SECRETS_VERSION=4.2.2 \
     GIT_LFS_VERSION=2.6.1 \
-    JNLP_JAVA_OPTIONS="-XX:+UseG1GC -XX:+ExplicitGCInvokesConcurrent -XX:+ParallelRefProcEnabled -XX:+UseStringDeduplication -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions -Dsun.zip.disableMemoryMapping=true"
+    JNLP_JAVA_OPTIONS="-XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90"  
```

Comment:
Settings are ok, but IMO you should rather set it again like:
`JAVA_GC_OPTS="-XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90"`
**See**
https://github.com/openshift/jenkins/blob/8b75b11f7dbcaefbed6161965c17da4d13c69ab2/slave-base/contrib/bin/run-jnlp-client#L183

If you overwrite `JNLP_JAVA_OPTIONS` that might overwrite other settings as well.
**See:**
https://github.com/openshift/jenkins/blob/8b75b11f7dbcaefbed6161965c17da4d13c69ab2/slave-base/contrib/bin/run-jnlp-client#L202

Reply:
@oalyman I have changed the param in Jenins agent to use JAVA_GC_OPTS insteead of JNLP_JAVA_OPTIONS